src/modules/FSXAProxyApi.spec.ts

```diff
@@ -65,20 +64,33 @@ describe('FSXAProxyAPI', () => {
       expect(locale).toEqual(actualRequestLocale)
     })
 
-    it('should throw a not found error when the response is 404', () => {
-      fetchMock.mockResponseOnce('', { status: 404 })
-      const actualRequest = proxyApi.fetchElement({ id, locale })
-      return expect(actualRequest).rejects.toThrow(FSXAApiErrors.NOT_FOUND)
+    it('should throw a not found error when the response is 404', async () => {
+      fetchMock.mockResponseOnce('', { status: HttpStatus.NOT_FOUND })
+      try {
+        await proxyApi.fetchElement({ id, locale })
+      } catch (error: any) {
```

Comment:
Should error be an instance of HttpError

Reply:
Unfortunately, the .toBeInstanceOf() function recognizes the error to be an instance of Error, because in the constructor super() is called. So we can't really check for that and I haven't found another way to check. If you know of one, please let me know.